src/mount-support.c

```diff
@@ -293,3 +294,58 @@ void setup_slave_mount_namespace()
 		die("can not make make / rslave");
 	}
 }
+
+#define BIND_MAX_LINE_LENGTH	512	// arbitrary
```

Comment:
Thanks, this is actually a leftover from the previous iteration of the code, removed now.